src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -38224,7 +38385,6 @@ static class E
     }
 }
 """;
-        // Tracked by https://github.com/dotnet/roslyn/issues/76130 : there should be no error on nameof
```

Comment:
>// Tracked by https://github.com/dotnet/roslyn/issues/76130 : there should be no error on nameof [](http://example.com/codeflow?start=8&length=96)

It is not obvious why this comment is removed.

Reply:
Talked to Mads and confirmed this is expected. Instead we want `object.P` to work (see comment below in this test).